OA-PASS/pass-docker
# Use usernames and passwords for ActiveMQ

## Overview
* Add the ability to configure fcrepo and indexer images for activemq
usernames and passwords, when desired
* Align's with Spring Boot conventions for variable names `SPRING_ACTIVEMQ_BROKER_URL`, `SPRING_ACTIVEMQ_USER`, and `SPRING_ACTIVEMQ_PASSWORD`
* Configures activemq to require authentication
* Bumps `fcrepo` and `indexer` images accordingly

## To test
* `docker-compose pull` followed by `docker-compose up`
* Log into [http://localhost:8161/admin/connections.jsp](http://localhost:8161/admin/connections.jsp) (admin/admin), and make sure there are four connections
